hack/community/readme.md

```diff
@@ -1,45 +1,24 @@
 # Cut a new community release
-The generate.sh script automates the community release process by copying over 
-the latest WMCO manifests to a new community-WMCO folder while also replacing 
-necessary fields such as the name, description, and image. It also generates a 
-signed commit ready to be pushed to community-operators-prod repo. 
+The generate.sh script is triggered to set up for the aws-e2e-operator-test 
```

Comment:
`is triggered to set up for the aws-e2e-operator-test` not sure what you mean by this

Reply:
I had trouble wording this. I'll change it to something like: the generate.sh script is called in the pre-step for the aws-e2e-operator-test.